editors/code/src/extension.ts

```diff
@@ -129,3 +135,43 @@ export function deactivate(): Thenable<void> {
     }
     return Server.client.stop();
 }
+
+async function askToCargoWatch() {
```

Comment:
Should I move it to runnable.ts? It doesn't seem to *exactly* fit there but maybe it makes more sense since it's task-related and whatnot.

Reply:
Yeah, I think `runnable.ts` is maybe a good place. 

Overall, I am not too happy with the current organization of code of the extension: there are way to many files; However, it is tolerable, so I am constantly postponing reorganizing it :)

Reply:
Right, makes sense :stuck_out_tongue: Moved this to `runnable.ts` then and renamed slightly